util/x509.go

```diff
@@ -0,0 +1,23 @@
+package util
+
+import (
+	"errors"
+	"fmt"
+	"os"
+)
+
+func ResolveRootCACertFile() (string, error) {
+	if rv, ok := os.LookupEnv("SSL_CERT_FILE"); ok {
+		return rv, nil
+	}
+	for _, potFile := range certFiles {
+		if _, err := os.Stat(potFile); err != nil {
+			if os.IsNotExist(err) {
+				continue
+			}
+			return "", fmt.Errorf("unexpected error resolving cert file: %w", err)
+		}
+		return potFile, nil
+	}
+	return "", errors.New("please set SSL_CERT_FILE to use --ssl-cert-file option")
```

Comment:
Nothing in this function relates to the cli option. Buildah might be used via the API as well so the cli options should generally not be used in the backend as these errors would make no sense to callers.

Reply:
Thanks that makes sense. Have reworded the message.